libappstream-builder/asb-utils.c

```diff
@@ -441,8 +441,10 @@ asb_utils_write_archive (const gchar *filename,
 	struct stat st;
 
 	a = archive_write_new ();
-	if (g_str_has_suffix (filename, ".gz"))
+	if (g_str_has_suffix (filename, ".gz")) {
 		archive_write_add_filter_gzip (a);
+                archive_write_set_filter_option (a, "gz", "timestamp", NULL);
+	}
 	if (g_str_has_suffix (filename, ".bz2"))
 		archive_write_add_filter_bzip2 (a);
```

Comment:
At least in libarchive the bz2 and xz filter have no timestamp option. *if* these formats have a timestamp(but I think they don't), these filters had to be enhanced in libarchive first.

Setting these options anyway would create an warning in libarchive.